packages/wp-dashboard/src/components/layout/index.js

```diff
@@ -22,11 +22,12 @@ import { InterfaceSkeleton } from '@web-stories-wp/dashboard';
  * Internal dependencies
  */
 import { useAdminSubMenu } from '../../effects';
+import EditorSettings from '../editorSettings';
 
 function Layout() {
   useAdminSubMenu();
 
-  return <InterfaceSkeleton />;
+  return <InterfaceSkeleton editorSettings={<EditorSettings />} />;
```

Comment:
WDYT about something more generalized, e.g.

```suggestion
  return <InterfaceSkeleton additionalRoutes={[{ route: '/editor-settings', component: EditorSettings }]} />;
```

Reply:
That's a good idea.